inc/search.class.php

```diff
@@ -3892,6 +3943,44 @@ static function addDefaultWhere($itemtype) {
       return $condition;
    }
 
+   /**
+    * Build parent condition for ITILFollowup, used in addDefaultWhere
+    *
+    * @param array $params keys: itemType, target, userTable, groupTable
+    *
+    * @return string
+    */
+   public static function buildITILFollowupParentCondition($params) {
+      $itemType   = $params['itemType'];
+      $target     = $params['target'];
+      $userTable  = $params['userTable'];
+      $groupTable = $params['groupTable'];
+
+      if (!Session::haveRight($itemType, $itemType::READALL)) {
+         if (Session::haveRight($itemType, $itemType::READMY)) {
+            // Case 1: can see only owned items
+            $user   = Session::getLoginUserID();
+            $groups = "'" . implode("','", $_SESSION['glpigroups']) . "'";
+
+            // Avoid empty IN ()
+            if ($groups == "''") {
+               $groups = '-1';
+            }
+
+            $subQueryUser = "SELECT `$target` from `$userTable` WHERE `users_id` = '$user'";
```

Comment:
Conditions are more complex on tickets (see https://github.com/glpi-project/glpi/pull/6150/files#diff-a801c52129f44d50ab6d1db9f049c4b7R3733 )

Reply:
Need tickets rights check are live, is this OK for you ?